src/main/java/cascading/tap/hive/HivePartitionTap.java

```diff
@@ -123,6 +160,91 @@ public void sourceConfInit( FlowProcess<? extends Configuration> flowProcess, Co
     super.sourceConfInit( flowProcess, conf );
     }
 
+
+  @Override
+  public String[] getChildPartitionIdentifiers( FlowProcess<? extends Configuration> flowProcess, boolean fullyQualified ) throws IOException
+    {
+    Configuration conf = flowProcess.getConfig();
+    IMetaStoreClient metaStoreClient = null;
+    try
+      {
```

Comment:
I forgot about the acid support. Sticking to 1.0 is fine. If people still use older versions of Hive, they can use an older release of cascading-hive. We just have to document this better (TODO for me).